### PR TITLE
Replace ember-cli-tailwind with postcss and tailwindcss

### LINKED ISSUE #407: Custom brand color no longer works in Ember CLI 3.13
I just upgraded my addon from Ember CLI 3.12 to 3.13 and the custom brand color is no longer working. It seems there's something strange going on in the CSS backing it.

This screenshot is on Ember CLI 3.12 (works):
Screen Shot 2019-10-07 at 4 19 46 PM

And here is Ember CLI 3.13 (doesn't work):
Screen Shot 2019-10-07 at 4 21 03 PM

### PR DESCRIPTION
**Pull request by josemarluedke**
Fixes #407. 

cc/ @samselikoff @rwwagner90 

@samselikoff Thanks for the video on EmberMap on this migration, that really helped.

### CODE REVIEW

**Comment by rwwagner90** [comment]
Thanks so much for the PR @josemarluedke! @samselikoff let me know if everything looks okay to you, and we can merge

File: addon/styles/components/_docs-brand-colors.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+  We use CSS Custom Properties so addons can customize their brand theme, but we need to provide a fallback for browsers that don't support them yet (IE). This is the way we do it.
+
+  If we get access to the PostCSS pipeline in the future we could use a plugin like this: https://github.com/postcss/postcss-custom-properties
```
Comment (samselikoff) [inline addon/styles/components/_docs-brand-colors.scss]: 👀 look at that! Should be able to use this now (future PR)

**Comment by samselikoff** [comment]
Awesome work @josemarluedke! Thanks for doing all this work!

Future PR idea: wonder if there's a way for us to distribute just the `dist` of this tailwind build into the final host apps. No reason for docs apps to have to pay the build-time cost of compiling this every time. Would be amazing to just ship the built (+ even purged!) tailwind docs styles.

**Comment by alexlafroscia** [comment]
> wonder if there's a way for us to distribute just the dist of this tailwind build into the final host apps

This would also solve the issue of #406, where the addon's instance of Tailwind and the `ember-cli-addon-docs` instance are conflicting.

**Comment by nadnoslen** [comment]
Thank you for this!
thanks

**Comment by ef4** [comment]
It would be good to get this released. I think without it addon-docs is generating invalid CSS for anyone on ember-cli >= 3.13.

**Comment by NullVoxPopuli** [comment]
I think this PR broke default styles. 
See: https://github.com/ember-learn/ember-cli-addon-docs/issues/453

**Pull request merged**